src/main/resources/swagger/api-docs.yaml

```diff
@@ -168,6 +168,36 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorReport'
+    post:
+      tags:
+        - Admin
+      summary: Gets a list of users for a list of Sam User IDs
+      operationId: adminGetUsersByIDs
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                type: string
+      responses:
+        200:
+          description: list of matching users
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/User'
+        403:
```

Comment:
I think you also need a block for a `400` response here (if someone requests > 1000 users)